openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -37,20 +32,24 @@ def process(self, instance):
             self.log
         )
 
-        preset = lib.load_capture_preset(data=capture_preset)
+        # Create temp directory for thumbnail
+        # - this is to avoid "override" of source file
+        dst_staging = tempfile.mkdtemp(prefix="pyblish_tmp_")
+        self.log.debug(
+            "Create temp directory {} for thumbnail".format(dst_staging)
+        )
+        # Store new staging to cleanup paths
+        filename = "{0}".format(instance.name)
```

Comment:
Isn't this just `filename = instance.name`?

Reply:
yes, same thing for the similar variable in extract_playblast